File: app/src/main/java/java/android/notes/listeners/IItemsClickListener.java
package java.android.notes.listeners;

import android.view.View;
import android.widget.ImageView;

public interface IItemsClickListener {


    void onOpenClick(int index);
    void onDeleteClick(int index, ImageView view);
    void onLongItemClick(View view);

    void removeNote(int index);
}

File: app/src/main/java/java/android/notes/listeners/IItemsClickListener.kt
package java.android.notes.listeners

import android.view.View
import android.widget.ImageView

interface IItemsClickListener {
    fun onOpenClick(index: Int)
    fun onDeleteClick(index: Int, view: ImageView?)
    fun onLongItemClick(view: View?)
    fun removeNote(index: Int)
}